danielMunteanu/src/main/java/Address.java

```diff
@@ -0,0 +1,26 @@
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getFullAddress() {
+        return "Full address is" + " " + "Country:" + this.country + " " + "City: " + this.city + " " + "City:" + this.street;
+
```

Comment:
many unused lines -> be compact -> make it beautiful

Reply:
will be fixed in next commit

Reply:
still waiting for it